src/Tvheadend.cpp

```diff
@@ -575,6 +575,14 @@ PVR_ERROR CTvheadend::AddTimer ( const PVR_TIMER &timer )
   uint32_t u32;
   dvr_prio_t prio;
 
+  if (timer.bIsRepeating && timer.iWeekdays)
+  {
+    if (m_conn.GetProtocol() > 17)
```

Comment:
I don't know about you but in cases such as this I think `>= 18` is clearer since it more clearly states that version 18 is the minimum required.